src/ReflectionExtension.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Parser Reflection API
+ *
+ * @copyright Copyright 2015, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\ParserReflection;
+
+use ReflectionExtension as BaseReflectionExtension;
+
+/**
+ * Returns AST-based reflections from extensions.
+ */
+class ReflectionExtension extends BaseReflectionExtension implements IReflection
+{
+    /**
+     * @var null|ReflectionClass[] ParsedReflection wrapped classes.
+     */
+    private $classes;
+
+    /**
+     * @var null|ReflectionFunction[] ParsedReflection wrapped functions.
+     */
+    private $functions;
+
+    /**
+     * Has extension been loaded by PHP.
+     *
+     * @return true
+     *     Enabled extensions are always loaded.
+     */
+    public function wasIncluded()
+    {
+        return true;
+    }
+
+    /**
+     * Returns list of reflection classes
+     *
+     * @return array|\ReflectionClass[]
```

Comment:
1. Please remove `array|`, because `[]` ending of `\ReflectionClass[]` already indicates to IDE that it's an array.
2. Shouldn't `\ReflectionClass[]` be `ReflectionClass[]` (namespaced version) of class instead?

Reply:
I will look into and play around with this. My recollection is that's the return type Scrutinizer extracted from the method from the parent class like the property initialized to an empty array of the type of it's members was specified. I don't recall if the issue came up here, or with the calling code. I will investigate further. :+1: